WcaOnRails/spec/i18n_locale_files_spec.rb

```diff
@@ -14,3 +14,14 @@
     end
   end
 end
+
+describe "Momentjs activation" do
+  let(:file_content) { File.read(Rails.root.join('app', 'assets', 'javascripts', 'application.js')) }
+  let(:locales) { I18n.available_locales - [:en] }
+
+  (I18n.available_locales - [:en]).each do |locale|
+    context "for " + locale.to_s do
+      it { expect(file_content).to include("//= require moment/#{locale}.js") }
```

Comment:
Right now the failure summary shows this:

> rspec ./spec/i18n_locale_files_spec.rb:24 # Momentjs activation for ru should include "//= require moment/ru.js"

Maybe change the beginning to use the `application.js` path?
"app/assets/javascripts/applications.js should activate momentjs for language and should include [...]"?